src/data-dictionary/events/K8sContainerSample/containerCpuCfsThrottledSecondsDelta.md

```diff
@@ -0,0 +1,9 @@
+---
+name: containerCpuCfsThrottledSecondsDelta
+type: attribute
+units:
```

Comment:
we know it's seconds, right? so we can add the unit for this?

Reply:
Yup! We fixed this!